src/components/shared/PhoenixbornPicker.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="flex flex-no-wrap">
+    <select
+      id="phoenixborn"
+      name="Phoenixborn"
+      class="border-2 bg-white border-gray-darker rounded h-full px-2 flex-auto"
+     :placeholder="placeholder"
+      @change="$emit('update:filter', $event.target.value)">
+      <option value="" disabled selected>{{ placeholder }}</option>
+      <option value="">All Phoenixborn</option>
+      <option value="james-endersight">Aradel Summergaard</option>
```

Comment:
Looks like a value copy/paste error. I think this stuff should probably be populated by the server, in any instance, since otherwise it's not possible to add new cards without updating the front-end source code. For instance, when a page with a PB filter is loaded for the first time, it could request a listing of Phoenixborn names/stubs from the Vuex store, and if that data isn't loaded run a quick `/v2/cards?type=phoenixborn` request, then cache the results in the store (that will also make it easy to support legacy Phoenixborn vs. Reborn ones).

Reply:
This is now done!
Not sure how much it's useful to have it cached though, as anything reload the page entirely. 
But it's a good thing to fetch the list dynamically!